RELEASE-NOTES.md

```diff
@@ -9,6 +9,9 @@ The project release numbers follow [Semantic Versioning](http://semver.org/spec/
 
 ## [Unreleased] - Release date yyyy-mm-dd
 
+### Changed
+- Updated Googletest to main from 04/13/2023. Commit: [12a5852e451baabc79c63a86c634912c563d57bc](https://github.com/google/googletest/commit/12a5852e451baabc79c63a86c634912c563d57bc)
```

Comment:
Please note the C++ version required for the new googletest, and that PGI is no longer supported (presumably, when configured with `ENABLE_GTEST`).

If any users require PGI, we might need to patch gtest to make this work.

Reply:
I asked the only internal user I knew about that used PGI and they had left the legacy PGI compiler and hadn't even tried the newer one ("nvhpc") yet. 

I say we go with not testing it until a later date.

Reply:
I agree, but we should note this in our `RELEASE-NOTES`.

Reply:
I added:

> Note: this version of Googletest requires C++14, and PGI is not supported. If you are using PGI, set ENABLE_GTEST OFF.

Reply:
I forgot to refresh the page. I can make note of this in the RELEASE-NOTES as well

Reply:
Thanks!